utils/unittest/blocks.go

```diff
@@ -0,0 +1,32 @@
+package unittest
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// RequireContainBlock is a test helper that fails if specified blocks list does not contain the
+// specified blocks.
+func RequireContainBlock(t *testing.T, block *flow.Block, blocks []*flow.Block) {
+	blockID := block.ID()
+	for _, b := range blocks {
+		if b.ID() == blockID {
+			return
+		}
+	}
+
+	require.Fail(t, fmt.Sprintf("block %x is not in the list %v", blockID, blocks))
+}
+
+// RequireContainBlock is a test helper that fails two blocks lists are not equal in their set of blocks.
+// It ignores the order of blocks in the lists.
+func RequireBlockListsMatchElements(t *testing.T, actual []*flow.Block, expected []*flow.Block) {
```

Comment:
I think we can replace these functions by composing some existing utility functions:

```go
RequireContainBlock(t, block, blocks)
```
can be replaced with:
```go
require.Contains(t, block.ID(), flow.GetIDs(blocks))
```



```go
RequireBlockListsMatchElements(t, actual, expected)
```
can be replaced with:
```go
require.Equal(t, flow.GetIDs(actual), flow.GetIDs(expected))
```
or, if we don't know that the order consistent:
```go
require.ElementsMatch(t, flow.GetIDs(actual), flow.GetIDs(expected))
```

This should also be `O(n)` rather than `O(n^2)` for the list comparison, and takes the hash out of the inner loop.

Reply:
Addressed in https://github.com/onflow/flow-go/pull/548/commits/5f594eb611a910249ad4348a2ac3d015fab512fc